gateway/src/main/java/io/pravega/example/iot/gateway/DataHandler.java

```diff
@@ -83,4 +93,41 @@ public String postData(@Context Request request, String data, @PathParam("remote
             throw e;
         }
     }
+
+    @POST
+    @Path("/rawData/{remoteAddr}")
```

Comment:
Rename remoteAddr parameter to routingKey.

Reply:
changed